### ISSUE TITLE
Suite Guide improvements

### ISSUE THREAD

**Issue by tomexx** [trezor/trezor-suite#3817]
Improvements of Suite Guide (Spec #3713, PR of the first version #3815)

Resolved in  #3859
- [x] preselect category according to the current route when reporting a bug
- [x] add nicer animations
- [x] F1 hotkey to toggle the visibility of the Guide

Resolved in #3949
- [x] add shadow under the top part of the Guide when contents of the Guide panel is scrolled down
- [x] emoji background
- [x] latest version check in bug reporting section
- [x] list what data are being shared when reporting a bug
- [x] feedback icon
- [x] hover mouse effects
- [x] analytics events 

Resolved in #3962
- [x] make loading of markdown articles dynamic **(after #3811 is done)**
- [x] remove hardcoded articles
- [x] use newest articles

**Comment by matejkriz** [comment]
- [ ] enable on mobile layout?

**Comment by tomasklim** [comment]
[] resize the feedback bar or open as a new page?

**Comment by tomexx** [comment]
> * [ ]  enable on mobile layout? by @matejkriz 
> * [ ]  resize the feedback bar or open as a new page within suite? by @tomasklim 

Good points! A design will be needed for this.

**Comment by sorooris** [comment]
Would it make sense to induce/open Guide with the F1 key (at least in the desktop version)?

**Comment by keraf** [comment]
> Would it make sense to induce/open Guide with the F1 key (at least in the desktop version)?

F1 is a common key for help in programs. I think it would be a great idea to map it, maybe even on web.

**Comment by glacierbits** [comment]
It would be nice to fix the gap. Although, I suspect we may need to do a bigger refactor to the scrollbars globally. 

<img width="291" alt="Screen Shot 2021-06-10 at 19 17 55" src="https://user-images.githubusercontent.com/29627086/121576811-b470fd00-ca20-11eb-8ffd-73391f93f7d8.png">

**Comment by tomasklim** [comment]
@matejzak Question regarding:

>  latest version check in bug reporting section

![Screenshot 2021-06-14 at 11 21 28](https://user-images.githubusercontent.com/33235762/121869776-ac57dc80-cd02-11eb-8184-5e9b275f721b.png)

Should **web** show `Up to date` or current version?

If the **desktop app** is not `Up to date` should it show just plain text version or also some link to update app modal?

The same question applies for the firmware version

**Comment by glacierbits** [comment]
I suggest to keep current versions (e.g. X.Y.Z) fro both App and FW to keep things simple.

Notifications should handle the CTA & warnings to update the app and FW.

**Comment by tomasklim** [comment]
Followup issue: #3992

Scrollbar issue: #3993

**Comment by sorooris** [comment]
**QA OK**

Resolved in #3859

- [x]  preselect category according to the current route when reporting a bug
- [x]  add nicer animations
- [x]  F1 hotkey to toggle the visibility of the Guide

 
Resolved in #3949

- [x]  add shadow under the top part of the Guide when contents of the Guide panel is scrolled down
- [x]  emoji background
- [x]  latest version check in bug reporting section
- [x]  list what data are being shared when reporting a bug
- [x]  feedback icon
- [x]  hover mouse effects
- [x]  analytics events

Resolved in #3962

- [x]  make loading of markdown articles dynamic (after Feat/guide data #3811 is done)
- [x]  remove hardcoded articles
- [x]  use newest articles

Suite: app 21.7.1 3d90909a
OS: macOS Big Sur 11.4
Device: model T 2.4.0

**Comment by wendys-cats** [comment]
**QA OK**
- am sad about the emoji background bit, it was not clear

**Info**:
**Suite** APP 21.7.1 3d90909aa68da250490f474b2d4cab02c0aad272
**OS**: Windows 10
**Device**: TT 2.4.1

WIP Improvements of Suite Guide (Spec #3713, PR of the first version #3815)

**Resolved in #3859**

 preselect category according to the current route when reporting a bug **OK**
 add nicer animations **OK**
 F1 hotkey to toggle the visibility of the Guide **OK**
 
**Resolved in #3949**

 add shadow under the top part of the Guide when contents of the Guide panel is scrolled down **OK** very pretty
 emoji background **OK** - am sad, we did not know which emojis this partained to
 latest version check in bug reporting section **OK** 
 list what data are being shared when reporting a bug **OK**
 feedback icon **OK**
 hover mouse effects **OK** on clickable elements
 analytics events 

**Resolved in #3962**

 make loading of markdown articles dynamic (after Feat/guide data #3811 is done) **OK**
 remove hardcoded articles **OK**
 use newest articles **OK**

**Comment by tomasklim** [comment]
> emoji background ?????

according to design, emoji in the feedback section should not have hover effect

**Comment by sorooris** [comment]
Created #4027 so we can approve this.
